frontend/packages/console-shared/src/utils/index.ts

```diff
@@ -0,0 +1,3 @@
+export function prefixedId(idPrefix, id) {
+  return idPrefix && id ? `${idPrefix}-${id}` : null;
```

Comment:
Under what circumstances would these values ever be null? Should we log a warning? I worry this might hide a bug if we silently handle it.

Again, not sure I agree with this helper :)